Import/Tests/AdditionalServiceExample.sql

```diff
@@ -0,0 +1,46 @@
+
+/***
+Declarations
+***/
+
+DECLARE @SupplierId varchar(6)
+DECLARE @ParentSolutionId varchar(16)
+DECLARE @AdditionServiceId varchar(14)
+DECLARE @ServiceName varchar(255)
+DECLARE @ServiceSummary varchar(300)
+DECLARE @ServiceDescription varchar(3000)
+DECLARE @Capabilities AdditionalServiceImportCapability
+
+/***
+SET DATA HERE
+***/
+
+SET @SupplierId = '10000'
```

Comment:
In our test data, notEmisWeb GP  is Solution Id 99999-89, SupplierId 99999. Can these please be adjusted to match the test data?

Reply:
Fixed